guides/main_concepts/indexes.md

```diff
@@ -1,152 +1,81 @@
 # Indexes
 
-An `index` is an entity, like a *table in SQL*, with a specific `schema` definition. It gathers a collection of `documents` with the structure defined by the schema.
+An index is the collection of a certain type of data.
 
-An **index is defined** by an unique identifier `uid` that is generated by MeiliSearch (*if none is given*) on [index creation](/references/indexes.md#create-an-index). It also has **a name** to help you track your different indexes.
+It is, like a table in `SQL`, or a collection in `mongoDb`, an entity that collects a set of documents.
 
-## Index UID and name
+An index is defined by an `uid` and contains the following information:
+- One document identifier <glossary word="attribute"/>
+- A set of relevancy rules (based on presets and customization)
+- A list of synonyms and stop-words
+- Rules for each field of a document
+- Settings
```

Comment:
For me the synonyms and stop-words were in settings, no?

Reply:
Yes, bad wording